<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>541</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxEdit">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>251</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>251</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditName"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string> 出生日期</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>  姓  名</string>
        </property>
        <property name="buddy">
         <cstring>name</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditId"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditSex"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditBirth"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>  性 别</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string> 病人编号</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>  病房号</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditDep"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxChart">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>220</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>查看阶段统计报表</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>查看图表</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxSearch">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>381</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Search</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>261</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,2">
      <property name="horizontalSpacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditSearch"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>160</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radio1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="add1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radio2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="add2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radio3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="add3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>860</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actCheck"/>
   <addaction name="actInsert"/>
   <addaction name="actDelete"/>
   <addaction name="actEdit"/>
   <addaction name="actConfirm"/>
   <addaction name="actAdd"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actCheck">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/images/CheckPics(1).jpg</normaloff>:/new/images/CheckPics(1).jpg</iconset>
   </property>
   <property name="text">
    <string>查看</string>
   </property>
   <property name="toolTip">
    <string>查看所有数据</string>
   </property>
  </action>
  <action name="actInsert">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/images/316.bmp</normaloff>:/new/images/316.bmp</iconset>
   </property>
   <property name="text">
    <string>插入数据</string>
   </property>
   <property name="toolTip">
    <string>插入数据</string>
   </property>
  </action>
  <action name="actDelete">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/images/318.bmp</normaloff>:/new/images/318.bmp</iconset>
   </property>
   <property name="text">
    <string>删除数据</string>
   </property>
   <property name="toolTip">
    <string>删除数据</string>
   </property>
  </action>
  <action name="actEdit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/images/103.bmp</normaloff>:/new/images/103.bmp</iconset>
   </property>
   <property name="text">
    <string>修改数据</string>
   </property>
   <property name="toolTip">
    <string>修改数据</string>
   </property>
  </action>
  <action name="actConfirm">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/images/704.bmp</normaloff>:/new/images/704.bmp</iconset>
   </property>
   <property name="text">
    <string>确定修改</string>
   </property>
   <property name="toolTip">
    <string>确定修改</string>
   </property>
  </action>
  <action name="actAdd">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/images/124.bmp</normaloff>:/new/images/124.bmp</iconset>
   </property>
   <property name="text">
    <string>添加查看</string>
   </property>
   <property name="toolTip">
    <string>添加查看</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
